tests/parm/rrfs_conus13km_hrrr.nml.IN

```diff
@@ -135,28 +132,11 @@
     grid_file = 'INPUT/grid_spec.nc'
 /
 
-
 &gfs_physics_nml
-    sfclay_compute_flux = @[SFCLAY_COMPUTE_FLUX]
     fh_dfi_radar = @[FH_DFI_RADAR]
     bl_mynn_edmf = 1
     bl_mynn_edmf_mom = 1
     bl_mynn_tkeadvect = .true.
-    rrfs_smoke = @[RRFS_SMOKE]
-    rrfs_smoke_debug = .false.
-    seas_opt = @[SEAS_OPT]
-    mix_chem = @[RRFS_SMOKE]
-    fire_turb = @[RRFS_SMOKE]
-    dust_opt = 5
-    drydep_opt = 1 
-    smoke_forecast = @[RRFS_SMOKE]
-    aero_ind_fdb = .false.
-    aero_dir_fdb = .false.
-    addsmoke_flag = 1
-    biomass_burn_opt = 1
-    wetdep_ls_opt = 1    
-    do_plumerise = @[RRFS_SMOKE]
-    plumerisefire_frq = 60
```

Comment:
If your code breaks smoke support, disabling the smoke test is not an acceptable way of fixing your code.